solr/contrib/hdfs/src/test/org/apache/solr/cloud/hdfs/HdfsTestUtil.java

```diff
@@ -91,6 +91,9 @@ public static MiniDFSCluster setupClass(String dir, boolean safeModeTesting, boo
     
     
     System.setProperty("solr.hdfs.blockcache.global", Boolean.toString(LuceneTestCase.random().nextBoolean()));
+
+    //FIXME: tells UpdateHandler to use HdfsUpdateLog without specifying such a class on UpdateLog in solrconfig.xml
+    System.setProperty("test.hdfs.forceHdfsUpdateLog", "true");
     
```

Comment:
Is this necessary now that UpdateLog initialisation is delegated to the DirectoryFactory?